robot/impl/resource_manager_modular_test.go

```diff
@@ -201,7 +202,7 @@ func TestModularResources(t *testing.T) {
 			Name:                "builtin",
 			API:                 motion.API,
 			Model:               resource.DefaultServiceModel,
-			ConvertedAttributes: &fake.Config{},
+			ConvertedAttributes: &motionBuiltin.Config{},
```

Comment:
I needed to change this to use the actual config the motoin builtin uses so that this condition doesn't occur: https://github.com/viamrobotics/rdk/pull/2879/files#diff-01573c2337c9ee8bafdab389eacebcbd7c16692bd71bec1208ead3665f40920fR123

I'm under the impression that the motion service should never be initialized with a config other than motion's builtin config. Please correct me if that impression is incorrect.